Final doc review (#58)
pasqal-io/qadence: docs/digital_analog_qc/pulser-basic.md

```

To convert the chain block into a pulse sequence, we define a `Register` with two qubits and combine it to create a circuit as usual. Then we construct a `QuantumModel` with a Pulser backend to convert it into a proper parametrized pulse sequence. Supplying the

> parameter values allows to sample from the pulse sequence result like any other `qadence` backend.

Reviewer: ```suggestion
parameter values allows to sample from the pulse sequence result like any other Qadence backend.
```

## Change device specifications

At variance with other backends, the Pulser one provides the concept of `Device`, inherited from
the Pulser. Check [this](https://pulser.readthedocs.io/en/stable/tutorials/virtual_devices.html) tutorial for more
information.

> A `Device` instance encapsulate all the properties defining a real neutral atoms processor, including but not limited

Reviewer: ```suggestion
A `Device` instance encapsulates all the properties for the definition of a real neutral atoms processor, including but not limited
```

    configuration={"device_type": Device.REALISTIC}
)

params = {
    "t": torch.tensor([383]),  # ns
    "y": torch.tensor([3*torch.pi/2]),
}

# sample from the result state vector and plot the distribution
sample = model.sample(params, n_shots=50)[0]
print(sample)
```

## Create your own gate

> A big advantage of using the block-based interface if `qadence` is that it makes it easy to create complex

Reviewer: ```suggestion
A big advantage of using the block-based interface if  Qadence is that it makes it easy to create complex
```
